backend/recruitmentapp/apps/api/v1/views.py

```diff
@@ -117,3 +118,40 @@ def create(self, request):
             data=serializer.data,
             status=status.HTTP_201_CREATED
         )
+
+
+class CompetenceViewSet(viewsets.ViewSet):
+    """Viewset for applicants and recruiters to view competences.
+
+    ### Permissions
+    All authenticated users.
```

Comment:
In the auto-generated documentation, this entire docstring is overriden by the docstrings of the action methods. This permissions bit needs to be repeated in the docstring of each method (`list()`, `retrieve()`...).